packages/skeleton-react/src/components/Tab/Tabs.tsx

```diff
@@ -0,0 +1,216 @@
+		const currIndex = tabs.indexOf(currTab);
+
+		// Determine the index of the next tab
+		let nextIndex = -1;
+		switch (event.code) {
+			case 'ArrowRight':
+				if (isRTL) {
+					nextIndex = currIndex - 1 < 0 ? tabs.length - 1 : currIndex - 1;
+					break;
+				}
+				nextIndex = currIndex + 1 >= tabs.length ? 0 : currIndex + 1;
+				break;
+			case 'ArrowLeft':
+				if (isRTL) {
+					nextIndex = currIndex + 1 >= tabs.length ? 0 : currIndex + 1;
+					break;
+				}
+				nextIndex = currIndex - 1 < 0 ? tabs.length - 1 : currIndex - 1;
+				break;
+			case 'Home':
+				nextIndex = 0;
+				break;
+			case 'End':
+				nextIndex = tabs.length - 1;
+				break;
+		}
+		if (nextIndex < 0) return;
+
+		// Set Active Tab
+		const nextTab = tabs![nextIndex!];
+		const nextTabInput = nextTab?.querySelector('input');
+		if (nextTabInput) {
+			nextTabInput.click();
+			(nextTab as HTMLDivElement).focus();
+		}
+	}
+
+    return (
+        <label
+            id={id}
+            className={`${base} ${rxActive} ${flex} ${background} ${border} ${text} ${padding} ${rounded} ${gap} ${cursor} ${classes}`}
+            aria-label={label}
+            title={title}
+        >
+            { /* NOTE: do not add additional classes to this <div> */ }
+            <div
+                className="size-full"
+                role="tab"
+                aria-controls={controls}
+                aria-selected={selected}
+                data-testid="tabs-control"
+                tabIndex={selected ? 0 : -1}
+                onKeyDown={onKeyDownHandler}
+                onKeyUp={onkeyup}
+            >
+                { /* Keep these classes on wrapping element */ }
+                <div className="h-0 w-0 flex-none overflow-hidden">
+                    <input ref={elemInputRef} type="radio" name={name} value={name} checked={selected} onChange={handleOnChange} onClick={onclick} tabIndex={-1} />
+                </div>
+
+                {children}
+            </div>
+        </label>
+    );
+};
+
+const TabsControlItem: React.FC<TabsControlItemProps> = ({
+    // Root
+    base = 'w-full',
+    flex = 'flex justify-center items-center',
+    gap = 'gap-2',
+    background = 'group-hover:preset-tonal-primary',
+    padding = 'p-2 px-4',
+    rounded = 'rounded',
+    classes = '',
+    // Children
+    children
+}) => {
+    return(
+        <div className={`${base} ${flex} ${gap} ${background} ${padding} ${rounded} ${classes}`}>
+            {children}
+        </div>
+    );
+};
+
+const TabsPanelItem: React.FC<TabsPanelItemProps> = ({
+    id = '',
+    value,
+    group,
+    // A11y
+    labelledBy,
+    // Root
+    classes = '',
+    // Children
+    children
+}) => {
+    return value === group && children && (
+        <div id={id} role="tabpanel" tabIndex={0} aria-labelledby={labelledBy} className={classes}>
+            {children}
+        </div> 
+    );
+};
+
+export const TabsControl = Object.assign(TabsControlRoot, { Item: TabsControlItem });
```

Comment:
I'm not sure we're following the dot notation convention properly.

- The root should be `<Tabs>`
- All children should be prefixed `<Tabs.Foo>`

You can see this in practice for Mantine:
https://mantine.dev/core/tabs/

Instead we have multiple prefixes, including:

`<Tabs.Foo>`, `<TabsList.Foo>`, and `<TabsControl.foo>`

Which results in:

```ts
import { Tabs, TabsList, TabsControl } from "...";
```

There should only be a single import item per tree of components components. Something like:

```ts
import { Tabs } from "...";
```
```html
<Tabs>
    <Tabs.List>
        <Tabs.Control>(labelText)</Tabs.Control>
        <Tabs.Control>(labelText)</Tabs.Control>
        <Tabs.Control>(labelText)</Tabs.Control>
    </Tabs.List>
    <Tabs.Panel>(panelContent)</Tabs.Panel>
    <Tabs.Panel>(panelContent)</Tabs.Panel>
    <Tabs.Panel>(panelContent)</Tabs.Panel>
</Tabs>
```

Additionally, the `<TabsControl.Item>` seems like a redundant abstraction. I'm assuming you were trying to replicate the structure for icon + label (like Svelte). However, icons should be passed via a prop. See the AccordionItem for reference.

Reply:
Feel free to ping me if we need to talk this one out! Given the scope of changes here I'll pause my review until these changes are implemented.